qemu/tests/cfg/drive_mirror.cfg

```diff
@@ -108,38 +108,22 @@
             reopen_timeout = 600
             variants:
                 - heavyload:
-                    download_link = http://weather.ou.edu/~apw/projects/stress/stress-1.0.4.tar.gz
```

Comment:
@xutian opened a new pull request for this part, please help to review.
https://github.com/avocado-framework/avocado-vt/pull/818